python/ray/_private/workers/default_worker.py

```diff
@@ -190,6 +190,15 @@
         "to import before accepting work."
     ),
 )
+parser.add_argument(
+    "--enable-resource-isolation",
+    type=bool,
+    required=False,
+    default=False,
+    help=(
+        "If true, core worker enables resource isolation through cgroupv2 by reserving resources for ray system processes."
```

Comment:
```suggestion
        "If true, core worker enables resource isolation by adding itself into appropriate cgroup."
```
Same comment as above.